### PR TITLE
Update files

### PR DESCRIPTION
**Pull request by maelle**
@sckott I actually think we could use most of https://raw.githubusercontent.com/r-lib/usethis/master/inst/templates/tidy-contributing.md?

Except we'd link to the chapter/section outlined in https://github.com/ropensci/dev_guide/issues/46 at the end.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
-* **issue_template.md** (see the [raw text](https://raw.githubusercontent.com/ropensci/dotgithubfiles/master/issue_template.md))
-* **pull_request_template.md** (see the [raw text](https://raw.githubusercontent.com/ropensci/dotgithubfiles/master/pull_request_template.md))
+* [**CONTRIBUTING.md**](dotgithub/CONTRIBUTING.md)
+* [**issue_template.md**](dotgithub/issue_template.md)
+* [**pull_request_template.md**](dotgithub/pull_request_template.md)
 
 Steps:
 
-* Put these three files in a `.github` directory in the base of your R package.
+* Put the [dotgithub folder](dotgithub/) containing these three files in a `.github` directory in the base of your R package.
```
Comment (sckott) [inline README.md]: we should specify to not put the `dotgithub` folder inside the `.github` folder, BUT instead the files within `dotgithub` within `.github`

**Review by sckott — CHANGES_REQUESTED** [review]
nice! just one thing

File: README.md (hunk 1 above)
Comment (maelle) [reply README.md]: rodev has a helper function now so one doesn't need to think but good point

**Comment by maelle** [comment]
I changed the phrasing, I'm gonna go ahead and merge 😸 Thanks!

**Pull request merged**